packages/transaction-controller/src/utils/validation.ts

```diff
@@ -194,3 +195,129 @@ function validateParamChainId(chainId: number | string | undefined) {
     );
   }
 }
+
+type GasFieldsToValidate = 'gasPrice' | 'maxFeePerGas' | 'maxPriorityFeePerGas';
```

Comment:
Minor, but should we try and keep all types at the top of the file above any functions?

Reply:
Done